### PR TITLE
W-15632241-PicklistMappingClarify-SG

### PR DESCRIPTION
**Pull request by sathya0**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: en/mulesoft/modules/ROOT/pages/ms_composer_using_actions_in_a_flow.adoc [hunk 1]
```diff
@@ -193,8 +193,8 @@ When mapping source data fields:
 
 * The source field must be a data pill with a fixed list of values.
 * The target field must be an enumerated xref:ms_composer_custom_expression_editor.adoc#data-types[data type] field or any string field. 
-* If any other unmapped value from the source system is found, the default mapping specifies the value in the target system . If you do not specify a default value, the original value from the source field is passed to the target field.
-* The mapping is stored at the organization level and can be reused across flows.
+* If any other unmapped value from the source system is found, the default mapping specifies the value in the target system. If you do not specify a default value, the original value from the source field is passed to the target field.
+* The mapping is stored at the organization level and can be reused across flows. You don't need to perform any extra steps when reusing mappings across different flows compared to reusing mappings within the same flow.
```
Comment (valkyrie69) [inline en/mulesoft/modules/ROOT/pages/ms_composer_using_actions_in_a_flow.adoc:197]: Can you shorten this to something like "You can reuse mappings both within the same flow and across different flows without performing extra steps." ?

**Review by valkyrie69 — APPROVED** [review]

File: en/mulesoft/modules/ROOT/pages/ms_composer_using_actions_in_a_flow.adoc (hunk 1 above)
Comment (sathya0) [reply en/mulesoft/modules/ROOT/pages/ms_composer_using_actions_in_a_flow.adoc:197]: What about this to compare that there are no extra steps if you want to reuse mappings across different flows?

No additional steps are required to reuse mappings across different flows compared to within the same flow.

**Review by valkyrie69 — DISMISSED** [review]

**Review by valkyrie69 — APPROVED** [review]
